examples/tutorial.dx

```diff
@@ -456,56 +456,56 @@ tableMean (for (i, j). (x.i.j, x.i.j))
   interface. Whereas `(/)` maps `a -> Float32 -> a` where `a` implements the
   `VSpace'` interface.
 
-' If we look in the Prelude, we can see that these interfaces are defined as:
+' If we look in the Prelude, we can see that these interfaces are defined as
+ (This will throw error because it mirrors the prelude, but we are just repeating it here.):
 
-interface Add' a
-  add' : a -> a -> a
-  sub' : a -> a -> a
-  zero' : a
+interface Add a
+  add : a -> a -> a
+  sub : a -> a -> a
+  zero : a
```

Comment:
Could you please briefly explaining dropping the trailing `'` from the declaration names here? I thought it was necessary to avoid "multiple definition" errors due to conflicts with Prelude declarations. Maybe that's not an issue anymore?

Reply:
I believe the `Multiple Definitions` error was coming from the type of tableMean

```haskell
def tableMean' [Add v, VSpace v] (x : n => v) : v =
```

which I changed to 

```haskell
def tableMean' [VSpace v] (x : n => v) : v =
```

VSpace implies Add. But not sure why that breaks it. 

Also I think you add the scaling in VSpace reversed (it's mul not div right?)

Reply:
I'm pretty sure re-defining a type class should also error. Can you please add this example to the [makefile tests](https://github.com/google-research/dex-lang/blob/main/makefile#L88-L92) so that we can check if it succeeds in CI?

Reply:
Added. Needed to download the files in the CI. hopefully that is okay.

Reply:
Hmmm this is a good question... I honestly don't know whether we can do that or not. The worst part is that the MNIST download page doesn't even mention a license, so I'm a bit reluctant to do so 😕 Could we use a different dataset by any chance?